pyproject.toml

```diff
@@ -27,9 +27,12 @@ PyYAML = "5.3.1"
 [tool.poetry.dev-dependencies]
 pytest = "7.1.3"
 
+[tool.poetry.group.dev.dependencies]
+mypy = "^1.8.0"
```

Comment:
Is it worth moving `pytest` into this group?

Reply:
Yes - I want to sort the deps out and get linting properly integrated, but trying to keep the PRs managable.